extra/alacritty.info

```diff
@@ -108,3 +108,5 @@ alacritty+common|base fragment for alacritty,
     hs, dsl=\E]2;\007, fsl=^G, tsl=\E]2;,
     Smulx=\E[4\:%p1%dm,
     Sync=\EP=%p1%ds\E\\,
+    XF, kxIN=\e[I, kxOUT=\e[0,
```

Comment:
```suggestion
    XF, kxIN=\E[I, kxOUT=\E[0,
```

Maybe put `XF` with other caps above, but I don't mind it being here?

Reply:
I like having all the things that go together on the same line. Makes it nice and easy to blame too.